app/views/customers/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 id='customer_name'>Customer Name: <%= @customer.name %></h1>
+
+
+<ul id="item_list">
+  <% @customer_items.each do |item| %>
```

Comment:
Here is where I would just do `@customer.items` and get rid of that instance variable in the controller.